dvc/tree/local.py

```diff
@@ -174,7 +174,7 @@ def move(self, from_info, to_info, mode=None):
         move(from_info, to_info, mode=mode)
 
     def copy(self, from_info, to_info):
-        tmp_info = to_info.parent / tmp_fname(to_info.name)
+        tmp_info = to_info.parent / tmp_fname("")
```

Comment:
It seemed helpful before to keep the relation between the original file name and this temp file if something goes wrong, but in reality the data is already in cache at this point so it would be helpful to just provide intructions/mechanism to recover it. Aaand we delete this temp file anyway down below :slightly_smiling_face: